docs/storage-abstraction.md

```diff
@@ -630,15 +574,15 @@ Put put =
         .build();
 ```
 
-##### Put with a condition
+##### Conditions for `Put`
```

Comment:
@Torch3333 Semi-related to [my previous question that you answered](https://github.com/scalar-labs/scalardb/pull/1119#discussion_r1339815316), I added the following paragraphs here since a description didn't exist. This is the same description from the [api-guide.md](https://github.com/scalar-labs/scalardb/blob/6f221b039243262a061aabd36f7652a17fcb5f75/docs/api-guide.md?plain=1#L731). Does this look OK? 

 I'm mostly wondering if the last sentence is correct since we don't mention `UnsatisfiedConditionException` elsewhere in this document.

Reply:
Thanks. We need to change the content since the behavior differs slightly for the storage abstraction.
Please refer to my other comment.